### ISSUE TITLE
Thoughts on Python3 support.

### ISSUE THREAD

**Issue by re-masashi** [youtube/spitfire#88]
This library seems to be really old. I was wondering if someone would be interested in Python3 support. If yes, I would make appropriate changes in the code and documentation.
Any response is appreciated.
Thanks in advance and regards.

**TL;DR:**
Shall I work on Python3 support?

**Comment by devfemibadmus** [comment]
yes i am very much interested

**Comment by re-masashi** [comment]
Thanks for showing interest.
Currently I have a [fork](https://github.com/re-masashi/spitfire) that works with Python3 and has appropriate benchmarks too.
Docs are available at [readthedocs](https://spitfire3.readthedocs.org).
Once again, thanks @devfemibadmus

**Comment by devfemibadmus** [comment]
https://pypi.org/project/spitfire3/ @re-masashi

**Comment by re-masashi** [comment]
@devfemibadmus yes, that's the project. Fiddle around a little bit and feel free to open issues or ask for any help.

**Comment by re-masashi** [comment]
Any updates or feedback? @devfemibadmus

**Comment by devfemibadmus** [comment]
still on it bro once there is i will pull request i am active

**Comment by devfemibadmus** [comment]
you can send me mail lets share contact devfemibadmus@gmail.com

**Comment by re-masashi** [comment]
Alright, surely will contact you.
